
AN ERROR HAS OCCURRED [the index 'a' does not exist]

CALLSTACK
*FUNCTION [__main__()] testData/static_analyzer/w275_complex.nut:12

LOCALS
[someBound] false
[sb] NULL
[ab] NULL
[bgs] NULL
[d] NULL
[bar] CLOSURE=FN:bar
[foo] CLOSURE=FN:foo
[vargv] ARRAY=[]
[this] NULL
